src/util/webhooks.ts

```diff
@@ -0,0 +1,30 @@
+import { getLogger } from "../config/logger";
+import statsd from "../config/statsd";
+import { metricWebhooks } from "../config/metric-names";
+
+export const getCurrentTime = () => new Date();
+
+export const calculateProcessingTimeInSeconds = (
+	webhookReceivedTime: Date,
+	webhookName: string,
+	status?: number,
+): number => {
+	const logger = getLogger("webhookProccessingTime");
+	const timeToProcessWebhookEvent =
+		(getCurrentTime().getTime() - webhookReceivedTime.getTime()) / 1000;
```

Comment:
Milliseconds would be fine here. We use milliseconds in other metrics so better to stay consistent

- [ ] use milliseconds instead of seconds